test/e2e/e2e_test.go

```diff
@@ -82,6 +64,7 @@ func init() {
 	flag.StringVar(&testContext.Host, "host", "", "The host, or apiserver, to connect to")
 	flag.StringVar(&testContext.RepoRoot, "repo-root", "../../", "Root directory of kubernetes repository, for finding test files.")
 	flag.StringVar(&testContext.Provider, "provider", "", "The name of the Kubernetes provider (gce, gke, local, vagrant, etc.)")
+	flag.IntVar(&testContext.MinStartupPods, "minStartupPods", 0, "The number of pods which we need to see in 'Running' state, before we try running tests. This is useful in any cluster which needs some base pod-based services running before it can be used.")
```

Comment:
nit: this is in 'Running' **phase with a 'Ready' condition of 'true'**

I wish we could find a better name for this... several folks misunderstand this as "the number of pods we wait for at startup" when really the behavior is "we _always_ wait for _all_ pods to be running & ready at startup; this is the minimum number we must find"